packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx

```diff
@@ -616,6 +631,8 @@ const DepositForm: React.FC<InjectedFormProps<{ form: string }, Props> & Props>
             height='48px'
             nature='primary'
             type='submit'
+            // @ts-ignore
```

Comment:
What's the ts error here?

Reply:
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/97299628/174159299-a1898fe9-4cc7-4e63-b5ac-fa6f87207ee3.png">

Reply:
I've tried with every every event type but it seems that the button props are closed, no clue why I can't add a simple `onClick`

Reply:
ohh, just got it, I can add the event on the submit function. Fixing it!